src/components/core/geoChart/tests/GeoChartLegendRenderer.spec.tsx

```diff
@@ -18,21 +18,39 @@ import {
     getExecutionResult,
 } from "../../../../../stories/data/geoChart";
 import { DEFAULT_COLORS } from "../../../visualizations/utils/color";
+import { IntlWrapper } from "../../base/IntlWrapper";
+import { IntlTranslationsProvider, ITranslationsComponentProps } from "../../base/TranslationsProvider";
+import { DEFAULT_LOCALE } from "../../../../constants/localization";
 
 interface ILegendFlags {
     hasSizeLegend?: boolean;
     hasColorLegend?: boolean;
 }
 
-function createComponent(customProps: IGeoChartLegendRendererProps) {
+function createComponent(customProps: Partial<IGeoChartLegendRendererProps>) {
     const legendProps = {
         ...customProps,
     };
-    const Wrapped = withIntl(GeoChartLegendRenderer);
+    const { colorLegendValue } = customProps;
+    const Wrapped = withIntl((props: Partial<IGeoChartLegendRendererProps>) => {
+        return (
+            <IntlWrapper locale={DEFAULT_LOCALE}>
+                <IntlTranslationsProvider>
+                    {(transplationProps: ITranslationsComponentProps) => (
+                        <GeoChartLegendRenderer
+                            {...props}
+                            colorLegendValue={colorLegendValue}
+                            numericSymbols={transplationProps.numericSymbols}
+                        />
+                    )}
+                </IntlTranslationsProvider>
+            </IntlWrapper>
+        );
+    });
```

Comment:
It could be shorter abit
```
const Wrapped = withIntl((props: Partial<IGeoChartLegendRendererProps>) => 
        (
            <IntlWrapper locale={DEFAULT_LOCALE}>
                <IntlTranslationsProvider>
                    {(transplationProps: ITranslationsComponentProps) => (
                        <GeoChartLegendRenderer
                            {...props}
                            colorLegendValue={colorLegendValue}
                            numericSymbols={transplationProps.numericSymbols}
                        />
                    )}
                </IntlTranslationsProvider>
            </IntlWrapper>
        );
    );
```